DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    experiments = list(df[df['experiment'].str.contains("025deg_jra55_iaf_omip2_cycle") &amp; \
</pre><p>Extremely minor, but I don't think you need to explicitly break a line with backslash, because the block is already defined by the brackets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>